src/topology/algebra/polynomial.lean

```diff
@@ -203,6 +203,31 @@ begin
     rw [←hcd, noroots, card_zero], },
 end
 
+lemma coeff_bdd_of_roots_le (B : ℝ) (d : ℕ) (f : F →+* K) :
```

Comment:
Same here, a docstring and some explanation will be more than welcome.

Reply:
Done. Thanks!